aten/src/ATen/WrapDimUtils.h

```diff
@@ -27,6 +28,24 @@ static inline int64_t maybe_wrap_dim(int64_t dim, int64_t dim_post_expr, bool wr
   return dim;
 }
 
+#ifndef __CUDA_ARCH__
+// on Windows, std::bitset cannot seem to cope with the
+// non-explicit half conversion in THCUNN/THCHalfAutoNumerics.cuh
+// so this is host-code only
+
+static inline std::bitset<64> dim_list_to_vector(IntList dims, int64_t ndims, bool wrap_scalar=true) {
```

Comment:
Make the `int` 64 a `size_t` should be okay. Try sth like 64UL or 64ULL on Windows. Or you can use a user-defined literal. Here is the [reference](https://stackoverflow.com/questions/22346369/initialize-integer-literal-to-stdsize-t/22346540).